### ISSUE TITLE
Support for device_name

### ISSUE THREAD

**Issue by jamonholmgren** [terriblelabs/motion-juxtapose#18]
Running motion-juxtapose with a different device name (say, `rake spec device_name='iPhone 6 Plus'`) causes failures:

```
ERROR: RuntimeError - Screenshots are different sizes
# RuntimeError: Screenshots are different sizes
```

**Comment by squidpunch** [comment]
@jamonholmgren interesting - I do this all the time for a 5s running 7.1 without harm (in fact my specs always run with this defined so my screenshots always match)

are you saying after having a 6 plus match, the next time they are still mismatched?

**Comment by jamonholmgren** [comment]
@squidpunch Just switching back and forth between `rake spec` and `rake spec device_name='iPhone 6 Plus'` causes juxtapose to fail, even if you match each time. At least I think this is the case -- I haven't tested fully. :-)

**Comment by squidpunch** [comment]
I was just digging around in here because I needed something for my app...and my inspect talk, ( #17 )

interesting - i wonder if 6 and 6 plus are reporting the same path (but shouldnt) 
I may be able to dig further on this if nobody else picks it up

**Comment by jamonholmgren** [comment]
I guess I should spin up a quick app to see if I can replicate it. :-)

**Comment by squidpunch** [comment]
yeah I bet thats what it is....

https://github.com/terriblelabs/motion-juxtapose/blob/master/lib/juxtapose/strategy/mac_bacon_strategy.rb#L16-L21
```
def device_name
      name = [UIDevice.currentDevice.model.gsub(/\s+Simulator/, '').downcase]
      name << 'retina' if UIScreen.mainScreen.scale > 1
      name << '5' if UIScreen.mainScreen.bounds.size.height > 480
      name.join('-')
end
```
so 5,6, and 6 plus are probably going to be saving screenshots as `iphone-5

**Comment by squidpunch** [comment]
I'm putting together a PR for this :)

**Comment by thegreatape** [comment]
Should be fixed in 0.3.0 by https://github.com/terriblelabs/motion-juxtapose/pull/19 — thanks, @squidpunch!

**Comment by jamonholmgren** [comment]
:+1:

**Comment by ohammersmith** [comment]
Works, except when in landscape. See #20.

**Comment by ohammersmith** [comment]
I included a fix for the orientation issue in my PR for #20.

**Comment by squidpunch** [comment]
fixed in #19
